frontend/src/pages/notebookController/NotebookController.tsx

```diff
@@ -28,16 +29,21 @@ export const NotebookController: React.FC = React.memo(() => {
     state.appState.user || '',
     state.appState.namespace || '',
```

Comment:
Should we not want to just update our namespace in our app to being the one in the config? Wouldn't that make the most sense?

Everything should act upon the same namespace, this configuration just changes where that is pointed... or am I misunderstanding the purpose behind this?

Reply:
No, the namespace here has to be the namespace where the dashboard runs, but the namespace in the dashboard config is where all notebooks/pvc/configmaps/secrets run, they are different and we need both.

Reply:
How hard would it be for you to rename all the `appState.namespace` references to something like `appState.dashboardNamespace` so we can distinguish between what the dashboard namespace is and what the target notebook (and accessories) namespace is?

I think it's important that we no longer refer to `namespace` so nonchalantly when it's got two possibilities

Reply:
Yeah, sure, I will rename it and rename all the references to it in case of any confusion.

Reply:
Updated.